sdk/identity/identity-vscode/CHANGELOG.md

```diff
@@ -6,14 +6,16 @@
 
 ### Breaking Changes
 
+- This package **is deprecated**. The method that this package used to extract the `Azure.Account` access token has been out of date and non-functional since Feb. 14, 2022. As an alternative, please consider [using `AzureCliCredential` to authenticate via the Azure CLI](https://github.com/azure/azure-sdk-for-js/tree/main/sdk/identity/identity#authenticate-via-the-azure-cli). In the future, if Visual Studio Code authentication becomes viable again, a new major version of this package or a separate package will be provided that implements that functionality.
```

Comment:
When you say `Azure.Account`, do you mean this extension? https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account

Reply:
Yeah. I had a latent memory that it was named that, but it should be "Azure Account" without the ".".